### ISSUE TITLE
Org Member Page - Manage Button

### ISSUE THREAD

**Issue by therealslimhsiehdy** [cisagov/manage.get.gov#2765]
### **[THIS IS A DRAFT, ASSIGNING TO ALYSIA TO REFINE -- WILL REMOVE THIS WHEN FINISHED]**

Focusing on the Manage button for the row expansion on the Org Member page

### Acceptance criteria

* Figma design here - [manage button](https://www.figma.com/design/xFtGLHVrhp0lvwh0gYWnbx/.gov-registar?node-id=8941-86507&node-type=frame)
- [ ] The "Manage" button should have the gear icon and Manage text next to it
- [ ] When clicked on, should open to display a new titled "Member access and permissions"
- [ ] Should also have breadcrumbs at the top for "Members > Manage members > " and the current page (not clickable) of "Member access and permissions"
- [ ] Display the title of "Member email" and the member's email
- [ ] Display the title of "Member access" and have a required radio button Y/N of Admin access vs Basic access. Please reference the Figma design for a clearer vision. [Confirm if we should have something pre-selected when coming to this page per permission state]
- [ ] Display the title of "Admin access permissions" and have two sections of "Organization domain requests" and "Organization members" along with required to be filled out radio buttons. Please reference the Figma design for a clearer vision. [Confirm if we should have something pre-selected when coming to this page per permission state]
- [ ] Have a "Cancel" and "Update member" button at the bottom. Reference Figma for the colour palette

**Comment by abroddrick** [comment]
Hey @gabydisarli or @AnnaGingle can either of you answer this: "Does "update member" redirect on save to [this Edit page](https://www.figma.com/design/xFtGLHVrhp0lvwh0gYWnbx/.gov-registar?node-id=8941-86507&node-type=frame&t=82c9BhKz7fZxsnxY-0)?"

**Comment by gabydisarli** [comment]
No, it goes back to their [Member profile ](https://www.figma.com/design/xFtGLHVrhp0lvwh0gYWnbx/.gov-registar?node-id=7375-20546&node-type=frame&t=EptaSlKjFNnWOTHZ-11)(Manage member page) with the success message at the top

**Comment by gabydisarli** [comment]
@zandercymatics No confirmation modal on edit permissions.
